views/options/settings.jade

```diff
@@ -138,12 +138,12 @@ script(type='text/ng-template', id='partials/options.settings.coupon.html')
   .container-fluid
     .row
       .col-md-6
-        h2 Coupon
+        h2 {{env.t('coupon')}}
         form.form-inline(role='form',ng-submit='enterCoupon(_couponCode)')
-          input.form-control(type='text', ng-model='_couponCode', placeholder='Enter Coupon Code')
+            input.form-control(type='text', ng-model='_couponCode', placeholder=env.t('couponPlaceholder'))
```

Comment:
Could you remove the extra spaces here and on line 146? Matters less here, but I'd like it to be consistent.